lib/openshift/project.rb

```diff
@@ -31,7 +31,7 @@ def visible?(user: nil, result: {}, quiet: false)
     alias exists? visible?
 
     def empty?(user: nil)
-      res = default_user(user).cli_exec(:status, v: true, n: name)
+      res = default_user(user).cli_exec(:status, suggest: true, n: name)
```

Comment:
@zhouying7780 how far back does `suggest` exist in the OCP release?

Reply:
@pruan-rht  I've checked 4.1 and 4.4, both have suggest.